package.json

```diff
@@ -32,6 +32,7 @@
     "aws-elasticsearch-connector": "^8.2.0",
     "aws-sdk": "^2.610.0",
```

Comment:
need ES or SDK?

Reply:
deleting ES. I was originally leaving SDK to allow for retrieving `IntTran URL` from param store. But since we're now getting the URL in `development` package, we'll no longer need the sdk here.